lib/utils/isnt-design-doc.js

```diff
@@ -0,0 +1,4 @@
+// Checks for a design doc, so we can filters out docs that shouldn't return in *All methods
+module.exports = function isntDesignDoc (row) {
+  return row.id.match(/^_design/) === null
```

Comment:
Just a side note but I think `return /^_design/.test(row.id)` [would be slightly faster](https://jsperf.com/test-vs-match-3) (depending on the browser).